Makefile

```diff
@@ -5,6 +5,8 @@
 CHECKOUTS = /var/cache/openssl/checkouts
 ##  Snapshot directory
 SNAP = $(CHECKOUTS)/openssl
+##  OTC data directory
+OMCDATA = $(CHECKOUTS)/data
```

Comment:
Comment says OTC, but the variable is `OMCDATA`

Reply:
Fixed